ortho_seq_code/tests/test_orthoseqs.py

```diff
@@ -1,74 +1,73 @@
 import numpy as np
-import pandas as pd
+from ortho_seq_code.orthogonal_polynomial import ortho_poly_command
```

Comment:
This should be imported with the other ortho_seq_code imports.

Reply:
you mean have both `from ortho_seq_code.orthogonal_polynomial import ortho_poly_command`
and `from ortho_seq_code.orthogonal_polynomial import orthogonal_polynomial` as it currently is? seems we gotta have both to get tests passing